src/ogdf/cluster/ClusterGraph.cpp

```diff
@@ -801,6 +801,9 @@ void ClusterGraph::delCluster(cluster c) {
 	}
 
 	clusters.del(c);
+#ifdef OGDF_DEBUG
```

Comment:
You might want to use `OGDF_HEAVY_DEBUG` since `consistencyCheck` has become quite large.